models/curvefi/ethereum/curvefi_ethereum_view_pools.sql

```diff
@@ -1,6 +1,8 @@
 {{ config(
     materialized = 'view',
     alias = 'view_pools',
+    materialized='table',
```

Comment:
materialized is used twice which causes a compilation error.

Reply:
Ah shoot, thanks for the pointer - I'll fix this!